chatbot-conversational_AI/README.md

```diff
@@ -0,0 +1,346 @@
+   ```
+   If you do not have an own Docker Hub account, you can also pull the built images from my account `lasseurban`.
+
+4. From the Command Line, deploy the bot secret:
+   > _NOTE:_ You only need to do this, if you have not already deployed the bot secret while deploying the bot observer tool in step 5.5. Please doublecheck now wether you have correctly deployed the bot secret. If you are unsure you can use the command `kubectl get secrets`. You should see the bot-secret of the Type `Opaque` with 10 data items.
+   
+   If you have not already done the deployment of the secret, go to step 5.5 and make the described adjustments in `k8s/secret.yaml`. Afterwards, use the following command to deploy the secret:
+   ```shell script
+   kubectl apply -f k8s/secret.yaml
+   ```
+
+5. Deploy the Kubernetes Job or the CronJob. Decide wether you want to update the chatbot immediately (use the job.yaml) or repetitively with a fixed schedule (use the cronjob.yaml). Both execute the same update-bot functionality.<br />
+   To use the simple **Kubernetes Job**, do the following steps:
+   1.  Within the `job.yaml` in the k8s directory, adjust the value of `spec.template.spec.containers.image`, commented with `#change it to your image`, to use your Docker image.
+   2. From the Command Line, deploy the Job:
+      ```shell script
+      kubectl apply -f k8s/job.yaml
+      ```
+   3. If you want to kill the job, use the following command:
+      ```shell script
+      kubectl delete -f k8s/job.yaml
+      ```
+   To use the **Kubernetes CronJob**, do the following steps:
+   1.  Within the `cronjob.yaml` in the k8s directory, adjust the value of `spec.template.spec.containers.image`, commented with `#change it to your image`, to use your Docker image.
+   2. In the same file, adjust the value of `spec.schedule`, commented with `#change it to your schedule`. Information on how you do that, can be found [here](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/). For example, you could set the schedule to "0 0 * * *" to update the bot every day at midnight.
+   3. From the Command Line, deploy the CronJob:
+      ```shell script
+      kubectl apply -f k8s/cronjob.yaml
+      ```
+   4. You can check the status of the CronJob:
+      ```shell script
+      kubectl get cronjobs
+      ```
+   5. If you want to kill the CronJob, use the following command:
+      ```shell script
+      kubectl delete -f k8s/cronjob.yaml
+      ```
+
+<br />
+
+## About the Contributer and the Project
+The development of this chatbot and the tutorial was a project contributed by Lasse Urban, who is a vocational training student at SAP. He spent one rotation in the STAR (Student Training and Rotation) Program with the Kyma Runtime Adoption team from April 2021 to October 2021. If you want to learn more about the project or the contributer, please visit [Lasse's GitHub page](https://github.com/lasseUrban/kyma-chatbot).
```

Comment:
Linking to personal profiles is not permitted as per SAP's policies. Please remove it.